### ISSUE TITLE
Multiple-image DR-GAN Training

### ISSUE THREAD

**Issue by MKYucel** [kayamin/DR-GAN#8]
Hi!

Great work here by the way, it's been really helpful. I have a couple of questions though;

I have tested this with my own dataset, with which I was able to get satisfactory results with single-image DR-GAN (in terms of the quality of the generated faces). However, in multiple-image case, I consistently observe generator diverging and all i get in generated images is noise. Have you had a similar experience related to the multiple image version, and if yes, what tricks did you do to alleviate this problem? One other question is that you say that images should be sequentially aligned for multi-image DR-GAN; have you observed performance drops when the data is not sequentially aligned (i.e. imagine having a sequence of 10 frames and randomly sampling N number of them)? 

Thanks very much in advance.

**Comment by wx176** [comment]
Hi @MKYucel How did you do with the single-image DR-GAN? I tried the single-image DR-GAN training, also the pre-trained model, but i didn't get right results. I followed the steps in Readme.md. Have you modified the code? And which dataset did you use? Did you use the cfp-dataset? Thanks!

**Comment by MKYucel** [comment]
Hi @wx176 , I have used my own proprietary dataset for both single and multi-image versions. I had to change my dataloader obviously. I also performed MTCNN alignment (which helped a bit) and labelled the data for the poses myself as well. I think following the README.md should suffice except from the occasional data preprocessing. Make sure you have a functioning data loader though, I myself lost a lot of time there.

**Comment by wx176** [comment]
Hello @MKYucel Thanks for your reply. I used the cfp data. In the demo folder, there is a ipynb notebook file. I followed the instructions in that file to do data preprocessing. Unfortunately, the result i obtained was totally wrong. The generated images were not like human faces, and the loss of the generator didn't converge. Now i am wondering if i could give a same pose value for profiles in different side.

**Comment by forthtemple** [comment]
@MKYucel, I have same problem. Single trains fine but not multi. I thought perhaps it was because in the paper they use a dataset with a lot of poses while the cfp dataset only has 2 poses.

**Comment by boomberung** [comment]
@MKYucel, hello, did u solve your problem?

**Comment by NBdemanong** [comment]
Hi @wx176. sorry to bother you.Now when i run this code, i also  find the loss of the generator didn't converge.so did you solved this problem? Thanks!
